### DISCUSSION TITLE
Apple M1 chip

### DISCUSSION THREAD

**Discussion by RazeeHJ** [nektos/act#1334] [Q&A]
Trying to test this but I am on Apple M1 Chip and receives this warning: 

WARN  ⚠ You are using Apple M1 chip and you have not specified container architecture, you might encounter issues while running act. If so, try running it with '--container-architecture linux/amd64'. ⚠

Not sure how to solve it?

**Comment by catthehacker** [comment]
It says so in the message....

> try running it with `--container-architecture linux/amd64`.

**Reply by PaulRBerg** [reply]
Having to always type `act --container-architecture linux/amd64` is cumbersome - is there no other way to get rid of this warning? Maybe it is possible to set the container architecture somewhere in the global settings, or maybe there is an environment variable for it?

**Reply by catthehacker** [reply]
`echo '--container-architecture linux/amd64' >> ~/.actrc`

**Reply by PaulRBerg** [reply]
Thank you.
